### PR TITLE
Add Google Tag Manager in admin header

### PR DESCRIPTION
**Pull request by laurajaime**
Deface has a strange fail due to `nokogiri` gem and the overrides with Deface don't work when applied inside the HEAD tag. See:

https://github.com/spree/deface/issues/84
https://github.com/spree/spree/issues/2633

### CODE REVIEW

**Review by tramuntanal — APPROVED** [review]
LGTM!

**Comment by laurajaime** [comment]
@tramuntanal can you review it again, please?

File: README.md [hunk 1]
```diff
@@ -30,4 +30,21 @@ npm install
...
 bin/rails s
-```
\ No newline at end of file
+```
+
+## Overrides, decorators and more.
+
+### Add Google Tag Manager to admin backoffice
+
+Deface has a strange fail with nokogiri gem and the overrides with Deface not works with this changes.
```
Comment (tramuntanal) [inline README.md:39]: Can you write the required `nokogiri` version in order to use deface in the HEAD tag please?

**Review by tramuntanal — COMMENTED** [review]
Ok, let's use these overrides but also add a test to check it in further upgrades

File: spec/features/remove_overrides_spec.rb [hunk 2]
```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+require "spec_helper"
+
+RSpec.describe "Overrides" do
+  it "remove overrides after upgrading to v0.26" do
+    # - Check header and application overrides views to set Google Tag Manager
```
Comment (tramuntanal) [inline spec/features/remove_overrides_spec.rb:7]:
```suggestion
    # - Check header and application overrides views to set Google Tag Manager (see #22 and "Add Google Tag Manager to admin backoffice" in the README 
```

**Review by tramuntanal — APPROVED** [review]
LGTM!

**Pull request merged**